gamestonk_terminal/fundamental_analysis/av_view.py

```diff
@@ -517,3 +517,120 @@ def clean_fundamentals_df(df_fa: pd.DataFrame, num: int) -> pd.DataFrame:
+                        "description": """Asset Quality Index measures the proportion of assets where potential benefit is less certain. A positive relation between AQI and earnings manipulation is expected."""
+                        }
+        ratios["SGI"] = {"raw": sales[0] / sales[1],
+                        "description": """Sales Growth Index shows the amount of growth companies are having. Higher growth companies are more likely to commit fraud so there should be a positive relation between SGI and earnings management."""
+                        }
+        ratios["DEPI"] = {"raw": (dep[1] / (ppe[1]+dep[1])) / (dep[0] / (ppe[0]+dep[0])),
+                        "description": """Depreciation Index is the ratio for the rate of depreciation. A DEPI greater than 1 shows that the depreciation rate has slowed and is positively correlated with earnings management."""
+                        }
+        ratios["SGAI"] = {"raw": (sga[0] / sales[0]) / (sga[1] / sales[1]),
+                        "description": """Sales General and Administrative Expenses Index measures the change in SG&A over sales. There should be a positive relationship between SGAI and earnings management."""
+                        }
+        ratios["LVGI"] = {"raw": (td[0] / ta[0]) / (td[1] / ta[1]),
+                        "description": """Leverage Index represents change in leverage. A LVGI greater than one indicates a lowe change of fraud."""
+                        }
+        ratios["TATA"] = {"raw": (icfo[0] - cfo[0]) / ta[0],
+                        "description": """Total Accruals to Total Assets is a proxy for the extent that cash underlies earnigns. A higher number is associated with a higher likelihood of earnings manipulation."""
+                        }
+        ratios["MSCORE"] = {"raw": (-4.84 + (0.92 * ratios["DSRI"]["raw"]) + (0.58 * ratios["GMI"]["raw"])
+                            + (0.404 * ratios["AQI"]["raw"]) + (0.892 * ratios["SGI"]["raw"])
+                            + (0.115 * ratios["DEPI"]["raw"]) - (0.172 * ratios["SGAI"]["raw"])
+                            + (4.679 * ratios["TATA"]["raw"]) - (0.327 * ratios["LVGI"]["raw"])
+                            ),
+                            "description": """The Beneish model uses financial ratios to check if it is likely that earnings are manipulated. A score of -5 to -2.22 indicated a low chance of fraud, a score of -2.22 to -1.78 indicates a moderate change of fraud, and a score above -1.78 indicated a high chance of fraud."""
+                            }
+
+        if ns_parser.sub:
+            for rkey, value in ratios.items():
+                if ns_parser.description:
+                    print(value["description"])
+                print(rkey + ": " + str(value["raw"]))
+        else:
+            if ns_parser.description:
+                print(ratios["MSCORE"]["description"])
+            print("MSCORE" + ": " + str(ratios["MSCORE"]["raw"]))
+
+    except ValueError:
```

Comment:
Can you add a newline (either \n to the string or just `print("")` after everything you print?